### PR TITLE
Multibracketvalidation

### PR DESCRIPTION
**Pull request by MerrybyPractice**

### CODE REVIEW

**Comment by MerrybyPractice** [comment]
- [x] Feature - balance brackets 

- [x] Readme.md

Unit Tests
- [x] Happy Path
- [x] - Expected failure
- [x] - Edge Case

File: src/main/java/MultiBracketValidation.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+public class MultiBracketValidation {
```
Comment (JessLovell) [inline src/main/java/MultiBracketValidation.java:1]: The ideal solution for this problem is to push all the brackets into a stack and as you come across a closing bracket, you should peek to see if the top is a matching closing bracket.

File: src/test/java/MultiBracketValidationTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        assertFalse(MultiBracketValidation.multiBracketValidation("}"));
+        assertFalse(MultiBracketValidation.multiBracketValidation("("));
+        assertFalse(MultiBracketValidation.multiBracketValidation(")"));
+        assertFalse(MultiBracketValidation.multiBracketValidation("[{}"));
+    }
+
+    //EdgeCases
+
+    @Test
+    public void testMultiBracketValidationEdgeCase() {
```
Comment (JessLovell) [inline src/test/java/MultiBracketValidationTest.java:33]: In this case it doesn't count as a feature.
